ZelBack/src/services/dockerService.js

```diff
@@ -592,7 +593,13 @@ async function appDockerCreate(appSpecifications, appName, isComponent, fullAppS
   const driverStatus = dockerInfoResp.DriverStatus;
   const backingFs = driverStatus.find((status) => status[0] === 'Backing Filesystem'); // d_type must be true for overlay, docker would not work if not
   if (backingFs && backingFs[1] === 'xfs') {
-    options.HostConfig.StorageOpt = { size: `${config.fluxapps.hddFileSystemMinimum}G` }; // must also have 'pquota' mount option
+    // check that we have quota
+    const hasQuotaPossibility = await deviceHelper.hasQuotaOptionForDevice('/var/lib/docker').catch((error) => {
```

Comment:
hasQuotaOptionForDevicethat function argument is device not path -> pls check code again u need use both functions

Reply:
Nice catch and Thank you for the fix